modules/standard/Path.chpl

```diff
@@ -134,6 +134,7 @@ proc absPath(name: string): string throws {
   :throws SystemError: Upon failure to get the current working directory.
 */
 proc file.absPath(): string throws {
+  use Path only;
```

Comment:
I'm not sure how I feel about needing a use of the same module to access its own symbols in a qualified manner.

Reply:
I suspect this change is not necessary as there are other compiler mods to support this pattern, and our discussion included it.

Reply:
Michael's right that this is not necessary anymore.  I'm guessing I added it in an early draft that wasn't working correctly yet, not noticing that it was self-referential (?).

Reply:
This wasn't the only instance of that that I noticed and I didn't look through the whole PR yet, btw.

Reply:
OK, I'll look around for others before PRing